src/mail_client.py

```diff
@@ -0,0 +1,35 @@
+import smtplib
+import time
+import getpass
+from src.mail_bomb import MailBomb
+
+
+class MailClient:
```

Comment:
to me it makes sense to make a client wrapper that does smtpobj operations often, so  i made a class.

Reply:
Well I am happy & impressed that you tried to add something special with class structure !